dask/dataframe/tests/test_shuffle.py

```diff
@@ -80,6 +80,19 @@ def test_shuffle_npartitions_task():
     assert set(map(tuple, sc.values.tolist())) == set(map(tuple, df.values.tolist()))
 
 
+def test_shuffle_npartitions_lt_input_partitions_task():
+    df = pd.DataFrame({"x": np.random.random(100)})
+    ddf = dd.from_pandas(df, npartitions=20)
+    s = shuffle(ddf, ddf.x, shuffle="tasks", npartitions=5, max_branch=2)
+    sc = s.compute(scheduler="sync")
```

Comment:
Any reason for the explicit `scheduler="sync"` here? Can we remove it?

Reply:
I just followed the other tests. No particular reason.  We can probably remove it   


https://github.com/dask/dask/blob/2b981ccf90c81997ebcc1a5f9576a7acb184b11a/dask/dataframe/tests/test_shuffle.py#L70-L74

Reply:
That makes sense.  We probably shouldn't require the single-threaded scheduler in any tests.  I'm sure every instance of that is a leftover debug statement, or someone doing exactly what you did :)  I know I'm guilty of both reasons.

Reply:
Makes sense. Removed from both with commit https://github.com/dask/dask/pull/7715/commits/86e6deac851cd702c106ba143460f0d5c4c2ce89

Reply:
Let me know if there is anything else missing. Thanks for the quick review.